photon-client/src/components/dashboard/CameraAndPipelineSelectCard.vue

```diff
@@ -232,6 +232,12 @@ useCameraSettingsStore().$subscribe((mutation, state) => {
       break;
   }
 });
+const wrappedCameras = computed<SelectItem[]>(() =>
```

Comment:
I see wrappedCameras a couple places - is there any way to consolidate? I only see it twice though which is good.